docs/introduction/faq-and-common-issues.md

```diff
@@ -26,6 +26,7 @@ For more detailed information about the Shopsys Framework, please see [Shopsys F
 - [How can I create Front-end Breadcrumb navigation?](#how-can-i-create-front-end-breadcrumb-navigation)
 - [Do you have any tips how to debug emails during development in Docker?](#do-you-have-any-tips-how-to-debug-emails-during-development-in-docker)
 - [Can I see what is really happening in the Codeception acceptance tests when using Docker?](#can-i-see-what-is-really-happening-in-the-codeception-acceptance-tests-when-using-docker)
+- [Why is there a faked PHP 7.2 platform in the Composer config?](#why-is-there-a-faked-php-72-platform-in-the-composer-config)
```

Comment:
I'm not sure about that fake. It's just specifying a fixed (or forced?) lower version which is guaranteed than is really used.

Reply:
I wasn't sure about that either, but I got that term from [the official Composer docs](https://getcomposer.org/doc/06-config.md#platform).

Reply:
Lets respect the composer docs